examples/regression/1-quickstart/plot_homoscedastic_1d_data.py

```diff
@@ -125,29 +126,58 @@ def plot_1d_data(
         ("linear", LinearRegression(fit_intercept=False)),
     ]
 )
+polyn_model_quant = Pipeline(
+    [
+        ("poly", PolynomialFeatures(degree=4)),
+        ("linear", QuantileRegressor(
+            alpha=1e-9,
+            fit_intercept=False,
+        )),
+    ]
+)
 
-Params = TypedDict("Params", {"method": str, "cv": int})
 STRATEGIES = {
-    "jackknife": Params(method="base", cv=-1),
-    "jackknife_plus": Params(method="plus", cv=-1),
-    "jackknife_minmax": Params(method="minmax", cv=-1),
-    "cv": Params(method="base", cv=10),
-    "cv_plus": Params(method="plus", cv=10),
-    "cv_minmax": Params(method="minmax", cv=10),
+    "jackknife": {"method": "base", "cv": -1},
+    "jackknife_plus": {"method": "plus", "cv": -1},
+    "jackknife_minmax": {"method": "minmax", "cv": -1},
+    # "cv": {"method": "base", "cv": 10},
+    "cv_plus": {"method": "plus", "cv": 10},
+    "cv_minmax": {"method": "minmax", "cv": 10},
+    "quantile": {"method": "quantile", "cv": "split"},
 }
 fig, ((ax1, ax2, ax3), (ax4, ax5, ax6)) = plt.subplots(
     2, 3, figsize=(3 * 6, 12)
 )
 axs = [ax1, ax2, ax3, ax4, ax5, ax6]
 for i, (strategy, params) in enumerate(STRATEGIES.items()):
-    mapie = MapieRegressor(
-        polyn_model, agg_function="median", n_jobs=-1, **params
-    )
-    mapie.fit(X_train.reshape(-1, 1), y_train)
-    y_pred, y_pis = mapie.predict(
-        X_test.reshape(-1, 1),
-        alpha=0.05,
-    )
+    if strategy == "quantile":
+        mapie = MapieQuantileRegressor(
+            polyn_model_quant, **params
+        )  # type: ignore
+        X_train, X_calib, y_train, y_calib = train_test_split(
```

Comment:
I would give `X_train` produced by `train_test_split` a different name to keep the "original" `X_train`. The figure of quantile conformalized regression does not have the same training point as other methods.

Reply:
So do we want to give the conformalized quantile regression additional points (for calibration) and train all the models with the same data? Otherwise, in order to make the graphs, we will need to add:
`X_train, y_train = X_split_train, y_split_train`